.github/workflows/ci.yml

```diff
@@ -8,7 +8,7 @@ jobs:
     steps:
       - uses: actions/checkout@v2
       - name: Build Docker image
-        run: docker build . --build-arg NF_IMAGE_VERSION=$GITHUB_SHA -t netlify/build:$GITHUB_SHA
+        run: docker build . --build-arg NF_IMAGE_VERSION=$GITHUB_SHA  --target build-image -t netlify/build:$GITHUB_SHA
```

Comment:
this will only run on one build image since xenial and focal are on separate branches. why do we need a target?

Reply:
oh nvm i see because test is a different stage and we wanna include it in build.